core/frontend/src/tile/map/MapTiledGraphicsProvider.ts

```diff
@@ -16,7 +16,10 @@ import { MapLayerImageryProvider, MapTileTreeReference, TiledGraphicsProvider, T
 /** Position of a map-layer in the viewport
  * @beta */
 export interface MapLayerIndex {
+  // true if layer is overlay, otherwise layer is background.
```

Comment:
convert to doc comments (`/** */`).

Reply:
Traditionally the API keeps the 'isOverlay' and 'index' parameters separated (i.e. `DisplayStyleState.mapLayerAtIndex`), but for `Viewport.getMapLayerIndexesFromIds` I needed to return an object containing both, so I created this interface. 

Now I realize that if  `MapLayerIndex` is not used in `MapLayerScaleRangeVisibility`, `MapLayerIndex` could be kept internal. 
What Do you think, better avoid exposing  `MapLayerIndex` ?

Reply:
Decided to make `MapLayerIndex ` internal and no longer reference it in `MapLayerScaleRangeVisibility`